es6-template-literals/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number}${street}`;
+
+    console.log('LearningFuze:', address);
+
+    const firstName = 'Uzair';
+    const lastName = 'Ashraf';
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log(`Bio: ${bio}`);
```

Comment:
One of the console logs is still different from its original version:

---

![image](https://user-images.githubusercontent.com/16143771/139783123-d672554e-ffc8-40d0-9f71-96a6baf2f263.png)